### PR TITLE
[Feature] Support rv1126 in sdk

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

support ssd-lite rknn model in MMDeploy SDK

## Modification

After #1203 

## BC-breaking (Optional)

NO

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: csrc/mmdeploy/net/rknn/rknn_net.cpp [hunk 1]
```diff
@@ -73,50 +128,75 @@ Result<void> RKNNNet::Init(const Value& args) {
...
   }
+  MMDEPLOY_DEBUG("model input num: {}, output num: {}", io_num.n_input, io_num.n_output);
+
+  auto get_tensor_shape = [](rknn_tensor_attr& attr) -> Result<TensorShape> {
+    TensorShape shape;
+    for (int i = 0; i < attr.n_dims; ++i) {
+      shape.push_back(attr.dims[i]);
+    }
+#ifdef RK_MODELS
+    //    return TensorDesc{device, data_type, shape, "#" + std::to_string(id)};
```
Comment (AllentDan) [inline csrc/mmdeploy/net/rknn/rknn_net.cpp:157]: may get removed

File: csrc/mmdeploy/net/rknn/rknn_net.cpp [hunk 2]
```diff
@@ -73,50 +128,75 @@ Result<void> RKNNNet::Init(const Value& args) {
...
+      MMDEPLOY_ERROR("MMDeploy SDK only supports RKNN-INT8 model");
+      return Status(eInvalidArgument);
+    }
+    input_attrs_.push_back(attr);
+    // Only support uint8 input data
+    OUTCOME_TRY(auto data_type, GetMMDeployDataType(RKNN_TENSOR_UINT8));
+    input_tensors_.emplace_back(
+        TensorDesc{device_, data_type, get_tensor_shape(attr).value(), "#" + std::to_string(i)});
   }
+  DebugRKNNTensorAttr("input", input_attrs_);
```
Comment (AllentDan) [inline csrc/mmdeploy/net/rknn/rknn_net.cpp:184]: seems it also works for release mode?

**Review by AllentDan — COMMENTED** [review]
Should dump-info and the doc for building with rknpu updated in #1203 or just in this PR?

File: csrc/mmdeploy/net/rknn/rknn_net.cpp (hunk 2 above)
Comment (lvhan028) [reply csrc/mmdeploy/net/rknn/rknn_net.cpp]: It prints the rknn tensor attribute, working for both release and debug mode. I changed the name to `PrintRKNNTensorAttr` to clarify the purpose.

**Comment by AllentDan** [comment]
CI failed.

**Review by lzhangzz — APPROVED** [review]

**Comment by AllentDan** [comment]
may add CI for rknpu cross-compiling.

File: csrc/mmdeploy/net/rknn/CMakeLists.txt [hunk 3]
```diff
@@ -2,19 +2,36 @@
...
 add_library(rknn SHARED IMPORTED)
 
-if(DEFINED ENV{RKNPU2_DEVICE_DIR})
+if (DEFINED ENV{RKNPU2_DEVICE_DIR})
     file(TO_CMAKE_PATH $ENV{RKNPU2_DEVICE_DIR} RKNPU2_DEVICE_DIR)
-else()
-    message(FATAL_ERROR "RKNPU2_DEVICE_DIR env must be defined")
-endif()
+endif ()
+if (DEFINED RKNPU2_DEVICE_DIR)
```
Comment (AllentDan) [inline csrc/mmdeploy/net/rknn/CMakeLists.txt:12]: Compiling failed for RKNPU if `RKNPU2_DEVICE_DIR` is also given in env variables.

**Review by AllentDan — APPROVED** [review]

**Review by AllentDan — APPROVED** [review]
LGTM again

**Pull request merged**